etcdctl/ctlv3/command/printer_table.go

```diff
@@ -25,6 +25,35 @@ import (
 
 type tablePrinter struct{ printer }
 
+func (tp *tablePrinter) Get(r v3.GetResponse) {
```

Comment:
Thanks @agargi - lgtm 
Table is not supported for other KV commands, I don't recall any discussion around it but I believe good to add this support starting with Get. Also, in general, the table and other format outputs are not streamlined but that's something we can do separately at later time. 
/cc @jingyih @gyuho